lib/logstash/filters/clone.rb

```diff
@@ -24,7 +30,7 @@ def register
   def filter(event)
     @clones.each do |type|
       clone = event.clone
-      clone.set("type", type)
+      ecs_support(clone, type)
```

Comment:
since these do different things I would rather split the logic into 2 methods 
(`set_event_type(event, type)`, `add_event_tag(event, tag)`) and so smt like:
```suggestion
      @event_enhance_method.(clone, type)
```
and have the "ecs setup" happen in initialize as for other plugins e.g.
```ruby
  def initialize(*params)
    super
    
    @event_enhance_method = method( ecs_select[disabled: :set_event_type, v1: :add_event_tag] )
  end
```